package.json

```diff
@@ -54,6 +54,7 @@
 	"dependencies": {
 		"@apollo/react-hooks": "^3.1.5",
 		"@chakra-ui/core": "^0.8.0",
+		"@chakra-ui/hooks": "^1.0.0-next.2",
```

Comment:
I noticed that in chakra they are using lately this package separately: https://github.com/chakra-ui/chakra-ui/blob/master/packages/tooltip/src/use-tooltip.ts#L1